tests/net/ethernet_mgmt/prj.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_ZTEST=y
+CONFIG_ZTEST_STACKSIZE=896
```

Comment:
Why not increase this to 1024 so that it won't fail immediately if something causes stack usage to increase?

Reply:
why not yes. I c/p what existed in some other test